src/js/app.js

```diff
@@ -3,6 +3,17 @@
 var bip39 = blocktrailSDK.bip39;
 var CryptoJS = blocktrailSDK.CryptoJS;
 
+var litecoinLatest = {
+    messagePrefix: '\x19Litecoin Signed Message:\n',
+    bip32: {
+        public: 0x019da462,
+        private: 0x019d9cfe
+    },
+    pubKeyHash: 0x30,
+    scriptHash: 0x32,
+    wif: 0xb0
+};
```

Comment:
Litecoin now has two script hash prefixes, but since the explorer API uses the latest one, and bitcoinjs the old one, we hard code the network.